src/main/java/org/betonquest/betonquest/commands/QuestCommand.java

```diff
@@ -118,6 +118,13 @@ public class QuestCommand implements CommandExecutor, SimpleTabCompleter {
 
     /**
      * Registers a new executor and a new tab completer of the /betonquest command.
+     *
+     * @param loggerFactory         logger factory to use
```

Comment:
This docs should deliver more information for what it is used,
like `logger factory to create new custum logger`.

Reply:
clearer documentation delayed until command rework